SemanticAutocomplete.md

```diff
@@ -0,0 +1,128 @@
+Semantic Autocomplete V2
+========================
+
+#### Table of Contents
+
+- [Summary](#summary)
+- [Request](#request)
+- [Response](#response)
+- [Example](#example)
+- [Versioning](#versioning)
+
+
+## Summary
+The Semantic Autocomplete service wraps the [Solr Suggester](https://lucene.apache.org/solr/guide/6_6/suggester.html) which is configured in DSAD's Semantic Search Solr boxes. Version 2 of the Semantic Autocomplete service is a complete rewrite of the original service and makes use of the latest FSTs provided to DSAD from Textkernel which are updated with improvements throughout the year. With the exception of some removed feilds from the response, this version of the service functions exactly like V1.
```

Comment:
I feel like this first sentence is relatively low level and maybe not of much use to potential users unless they have an in depth knowledge of Solr and our Semantic Search service. I think it would be better to describe just at a high level what it does and/or why they would want to use it.